README.rst

```diff
@@ -28,6 +29,9 @@ To run this package locally:
 
 To cleanup after a run, use either ``bin/ci_imsim_run.py --clean`` or ``rm -rf DATA/``.
 
+Note that there are 6 bands and 1 patch in `testdata_ci_imsim`, coadd-level processing will not 
+benefit from using over 6 cores, although visit-level and some coadd post-processing tasks will.
```

Comment:
Alternate wording suggestion, since I'm not sure this comment is helpful here: there are 30 detector-visits, so running with `-j30` wouldn't be unreasonable to drastically cut the single visit runtime. As I understand it, `-j` is not allocating processes, so specifying a larger number there wouldn't prevent other users from using those cores once the most-parallel portion has completed. How does the processing time compare between coadd and single visit?

```suggestion
Note that there are 30 detector visits across 6 bands and 1 patch in ``testdata_cim_imsim``, so running with at least ``-j 6`` will speed up coadd processing, while up to ``-j 30`` will reduce single frame runtime.
```

Reply:
There are actually 36 detector visits, but only 30 of them make it to consolidateVisitSummary. I'm not sure why as they were all selected to at least partly overlap the patch, although maybe there's enough masked out of 6 of them that they don't get included in coadds.

As for runtime, in my last two runs with `-j 6`, it took 25-30 minutes to get to assembleCoadd, then another 50 minutes (full 4k patch skymap)/14 minutes (small 2k patch skymap) to finish. Some of that time was on forcePhotCcd tasks running after assembleCoadd. So yes, it should benefit from more than 6 cores if you have plenty to spare.